publisher/middleware.py

```
@@ -0,0 +1,20 @@
+from django.utils import timezone
+from django.conf import settings
+import pytz
```

Ruff (0.12.2)

`pytz` imported but unused

Remove unused import: `pytz`

(F401)